ads/google/doubleclick.js

```diff
@@ -55,39 +55,21 @@ export function doubleclick(global, data) {
     };
   }
 
-  // Center the ad in the container.
-  const container = global.document.querySelector('#c');
-  setStyles(dev().assertElement(container), {
-    top: '50%',
-    left: '50%',
-    bottom: '',
-    right: '',
-    transform: 'translate(-50%, -50%)',
-  });
+  centerAd();
+
+  const url = getUrl(data);
+
+  whichGladeExperimentBranch(data, url, experimentFraction);
```

Comment:
Not a fan of this setup nor the name whichGladeExperimentBranch given it currently can write GPT!  Instead, I suggest the following:

centerAd();
const gptFilename = selectGptExperiment(data);
writeAdScript(data, gptFilename);

writeAdScript(data, gptFilename) {
  if (gptFilename || data.useSameDomainRenderingUntilDeprecated != undefined || data.multiSize) {
     doubleClickWithGpt(data, GladeExperiment.GLADE_OPT_OUT, `https://www.googletagservices.com/tag/js/${gptFilename || 'gpt.js'}`);
  } else {
      // random selection code here...
  }
}